app/internal/page/sign_your_lpa.go

```diff
@@ -0,0 +1,96 @@
+package page
+
+import (
+	"net/http"
+
+	"golang.org/x/exp/slices"
+
+	"github.com/ministryofjustice/opg-go-common/template"
+)
+
+type signYourLpaData struct {
+	App    AppData
+	Errors map[string]string
+	Lpa    *Lpa
+	Form   *signYourLpaForm
+}
+
+const (
+	CertificateProviderHasWitnessed = "cp-witnessed"
+	WantToApplyForLpa               = "want-to-apply"
```

Comment:
if you're going to pull these out then I think you should also inject them into the template to make the circle complete